configs/next/packages/valgrind/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="Valgrind"
 ATSRC_PACKAGE_VER=3.14.0
-ATSRC_PACKAGE_REV=d2af42d826d6
+ATSRC_PACKAGE_REV=331949c1c5cf
```

Comment:
OK.

This commit is actually after the release of 3.14.  Previous builds weren't.  :)

```
$ git describe 331949c1c5cf
VALGRIND_3_14_0-2-g331949c1c
```